src/FSharp.FGL.IO/GDF.fs

```diff
@@ -136,3 +136,84 @@ module GDF =
         path
         |> System.IO.File.ReadAllLines
         |> fromArray
+
+
+    //Returns the GDFValue-type as string
+    let typeOfGDFValue (value:GDFValue) =    
+        match value with
+            | VARCHAR     x   -> "VARCHAR"
+            | BOOLEAN     x   -> "BOOLEAN"
+            | DOUBLE      x   -> "DOUBLE"
+            | INT         x   -> "INT"      
+
+    //Applies typeOfGDFValue on all GDF Values of a given Map
+    let private getGDFValue (value:Map<'key,GDFValue>) =
+        let valueList = Map.toList value|>List.map(snd)
+        List.map(typeOfGDFValue) valueList
+
+    //Adds quotationmarks at the beginning and the end of strings
+    let private addQuotationsToString (x:string) : string =
```

Comment:
I think a `sprintf` would be better here. You could just ditch this function and use `sprintf "'%s'"` below